README.rst

```diff
@@ -93,24 +97,26 @@ Dropbox
 ~~~~~~~
 
 `Create an app on Dropbox <https://www.dropbox.com/developers/apps/create>`_,
-and store the app key and app secret in the file
-``secrets/dropbox-credentials.ini``, like this:
+and store the app key in the file
+``.env``, like this:
 
 .. code-block:: ini
 
-    [default]
-    dropbox_app_key=foo
-    dropbox_app_secret=bar
+    DROPBOX_KEY=foo
```

Comment:
As far as I know we are not using it. I tested my branch and I am able to upload without it.